cli/find.php

```diff
@@ -92,17 +94,12 @@
     cli_error(get_string('invalidcharacter', 'tool_advancedreplace'));
 }
 
-// Perform the search.
-$result = helper::search($search, !empty($options['regex-match']), $tables, $options['summary'] ? 1 : 0);
-
-// Notifying the user if no results were found.
-if (empty($result)) {
-    echo "No results found.\n";
-    exit(0);
-}
-
 // Start output.
-$fp = fopen('php://stdout', 'w');
+if (!empty($options['output'])) {
```

Comment:
I think its better to force --output as required because then we can use stdout for the progress bars

Reply:
Hi @brendanheywood I have made this as a required param